vfs/xts/xts.go

```diff
@@ -0,0 +1,275 @@
+package xts
+
+import (
+	"encoding/hex"
+	"io"
+
+	"github.com/ncruces/go-sqlite3"
+	"github.com/ncruces/go-sqlite3/internal/util"
+	"github.com/ncruces/go-sqlite3/vfs"
+	"golang.org/x/crypto/xts"
```

Comment:
```suggestion
	"golang.org/x/crypto/xts"

	"github.com/ncruces/go-sqlite3"
	"github.com/ncruces/go-sqlite3/internal/util"
	"github.com/ncruces/go-sqlite3/vfs"
```

Do you care about formatting matching `goimports -local github.com/ncruces/go-sqlite3`? If using `golangci-lint`, you'd add this to catch:

```yaml
linters-settings:
  goimports:
    local-prefixes: github.com/ncruces/go-sqlite3
```

Reply:
Thanks! See: ace01b2.